template/module/__init__.py

```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# © <YEAR(s)> <AUTHOR(s)>
```

Comment:
@dreispt 
Good point.
Maybe, should be a good idea use the same order in manifest key:
```python
    "website": "https://odoo-community.org/",
    "author": "<AUTHOR(S)>, Odoo Community Association (OCA)",
    "license": "AGPL-3",
    "application": False,
```
Just for satisfy our OCPD.